Podfile

```diff
@@ -169,7 +169,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.78.0'
+  gutenberg :commit => '214827b159cbeea00415ff30727388758e656c4e'
```

Comment:
**Before merging this PR,** we'll need to update this to a new 1.78.1 release tag.